### ISSUE TITLE
UWP - Relational data LINQ Queryable exception

### ISSUE THREAD

**Issue by mkieres** [realm/realm-dotnet#1568]
## Goals
Query relational data

## Expected Results
Retrieval of data

## Actual Results
Exception:
"System.IO.FileLoadException: 'Could not load file or assembly 'System.Linq.Queryable..."

## Code Sample
Brand new UWP project with Realm nuget packages. When debugging the app I'm getting an exception when trying to query relational data.

NOTE: Same code works fine on brand new Xamarin project (tested on UWP and Android)

[UWP_Queryable_Exception.zip](https://github.com/realm/realm-dotnet/files/1314304/UWP_Queryable_Exception.zip)

## Version of Realm and Tooling
* Realm Version: 1.6.0
* Environment: Windows 10 with VS 2017 15.3.4 Community

**Comment by nirinchev** [comment]
Can you try adding the [System.Linq.Queryable](https://www.nuget.org/packages/System.Linq.Queryable/) nuget? It should have been added as a dependency of Remotion.Linq, but it's possible NuGet has failed to do so. Additionally, you should remove the Realm NuGet from your project as that may cause issues with resolving the native library. As Sync is not yet supported for UWP, you only need the Realm.Database package.

**Comment by mkieres** [comment]
Hi nirinchev,

Thanks for your reply

I tried both suggestions. I removed Realm nuget dependency and added System.Linq.Queryable nuget (tried with versions 4.3.0, 4.0.1 and 4.0.0). I also double checked if System.Linq.Queryable libraries are being copied to the bin folder - they are. Unfortunately this did not help

There's seems to be an explicit dependency on the particular System.Linq.Queryable version, as the exception that I get goes as follows "Could not load file or assembly 'System.Linq.Queryable, Version=**4.0.20.0**".

Any other suggestions that I could try?

Find updated version of my sample attached

[UWP_Queryable_Exception_Updated.zip](https://github.com/realm/realm-dotnet/files/1320829/UWP_Queryable_Exception_Updated.zip)
